src/tools/singlejar/output_jar.cc

```diff
@@ -192,12 +193,19 @@ int OutputJar::Doit(Options *options) {
 
   for (auto &rdesc : options_->resources) {
     // A resource description is either NAME or PATH:NAME
-    std::size_t colon = rdesc.find_first_of(':');
+    // Find the last ':' instead of the first because Windows uses ':' as volume
+    // separator in absolute path.
+    std::size_t colon = rdesc.find_last_of(':');
     if (0 == colon) {
       diag_errx(1, "%s:%d: Bad resource description %s", __FILE__, __LINE__,
                 rdesc.c_str());
     }
-    if (std::string::npos == colon) {
+    bool shouldNotSplit = colon == std::string::npos;
```

Comment:
A negative boolean is difficult to understand ("if true, then don't do something").
Could you please flip it to be positive? ("If true, then split.")

Reply:
Done.